<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>100</x>
    <y>100</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SecuLite</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="proxy_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Proxy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="intruder_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Intruder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="repeater_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Repeater</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
